design-assets/README.md

```diff
@@ -0,0 +1,17 @@
+# CMSDS Design Assets
+
+In this folder you will find the following
+
+- CMSDS Sketch file
+- CMSDS fonts (Bitter, Icons, and Open Sans)
```

Comment:
I think Icons should be a separate line right?

Reply:
Technically the Icons.zip is FontAwesome... I'm thinking maybe we rename that to FontAwesome and Keep it as fonts

Reply:
ooh right, fontawesome is considered a font. I'm down with renaming it FontAwesome